sdk/src/test/tutorial/java/com/finbourne/lusid/tutorials/ibor/Transactions.java

```diff
@@ -130,4 +130,64 @@ public void load_cash_transaction() throws ApiException {
         assertEquals(1, transactions.getValues().size());
         assertEquals(transaction.getTransactionId(), transactions.getValues().get(0).getTransactionId());
     }
+
+    @Test
+    @LusidFeature("F18")
```

Comment:
Should this map to feature F31?

https://gitlab.finbourne.com/salesengineering/lusid-features-report/-/blob/master/features_sdk.md

Reply:
Oh yes, right you are!